com.bardsoftware.constable/src/main/kotlin/com/bardsoftware/constable/Main.kt

```diff
@@ -0,0 +1,14 @@
+package com.bardsoftware.constable
+
+import java.io.File
+
+fun main() {
```

Comment:
Pls add application plugin to `build.gradle` so that it could be launched with `gradle run`. You'll also need to add `settings.gradle`, otherwise it will not run in this GanttProject multi-project build environment